pkg/api/v1beta1/dynakube/properties.go

```diff
@@ -414,9 +414,14 @@ func (dk *DynaKube) Tokens() string {
 	return dk.Name
 }
 
-// TenantUUIDFromApiUrl gets the tenantUUID from the ApiUrl present in the struct, if the tenant is aliased then the alias will be returned.
-func (dk *DynaKube) TenantUUIDFromApiUrl() (string, error) {
-	return tenantUUID(dk.Spec.APIURL)
+func (dk *DynaKube) TenantUUIDFromConnectionInfo() (string, error) {
```

Comment:
```suggestion
func (dk *DynaKube) TenantUUID() (string, error) {
```

as there is no other `TennantUUIDFromXYZ` function, just having it with a shorter name would be better